core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java

```diff
@@ -77,8 +71,9 @@ public Status getStatus()
 	public static DiscloseRequestMessageDTO createInstance(UserDTO actingUser, DiscloseRequestMessage messageEntity)
 	{
 		GoalConflictMessage targetGoalConflictMessage = messageEntity.getTargetGoalConflictMessage();
-		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), messageEntity.getNickname(),
-				messageEntity.getStatus(), targetGoalConflictMessage.getOriginGoalConflictMessageID(),
+		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), actingUser,
```

Comment:
You use `actingUser` as a parameter, while the other subclasses of `BuddyMessageDTO` use `UserDTO.createInstanceIfNotNull(messageEntity.getUser())`. Could you please make these uniform? (See `BuddyConnectRequestMessageDTO.createInstance`, `BuddyConnectResponseMessageDTO.createInstance`, `BuddyDisconnectMessageDTO.createInstance`.) If the user is deleted, does `actingUser` not become `null`? The message factory may construct these messages (by an inbox request) after the related user has been deleted.